ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/api/LedgerEntry.scala

```diff
@@ -29,13 +27,6 @@ object LedgerEntry {
       val envelope: Bytes
   ) extends LedgerEntry
 
-  /**
```

Comment:
This additional indirection of `LedgerEntry` -> `LedgerRecord` is no longer required (it was only introduced to allow  `Heartbeat` entries as well).

Reply:
This will be fixed in https://github.com/digital-asset/daml/pull/5383.